app/src/main/java/com/example/mykotlincalculator21/mvp/model/CalculatorModel.kt

```diff
@@ -1,52 +1,129 @@
 package com.example.mykotlincalculator21.mvp.model
 
 import com.example.mykotlincalculator21.mvp.contract.CalculatorContract
-import com.example.mykotlincalculator21.utils.EMPTY_STRING
-import com.example.mykotlincalculator21.utils.ERROR_MESSAGE
-import com.example.mykotlincalculator21.utils.OPERATOR_DIVIDE
-import com.example.mykotlincalculator21.utils.OPERATOR_MINUS
-import com.example.mykotlincalculator21.utils.OPERATOR_MULTIPLY
-import com.example.mykotlincalculator21.utils.OPERATOR_SUM
+import com.example.mykotlincalculator21.utils.ResultUtils
+import com.example.mykotlincalculator21.utils.NumbersUtils.POSITION_ONE
+import com.example.mykotlincalculator21.utils.NumbersUtils.POSITION_ZERO
+import com.example.mykotlincalculator21.utils.NumbersUtils.ZERO_NUMBER_DOUBLE_TYPE
+import com.example.mykotlincalculator21.operand.Operand
+import com.example.mykotlincalculator21.utils.StringUtils.EMPTY_STRING
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_DIVIDE
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_MINUS
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_MULTIPLY
+import com.example.mykotlincalculator21.utils.StringUtils.OPERATOR_SUM
 
 class CalculatorModel : CalculatorContract.CalculatorModelContract {
-    private var firstOperand: String = EMPTY_STRING
-    private var secondOperand: String = EMPTY_STRING
+    private val firstOperandUtils = Operand()
+    private val secondOperandUtils = Operand()
     private var operator: String = EMPTY_STRING
     private var result: String = EMPTY_STRING
+    private var error = ResultUtils.SUCCESS
```

Comment:
why the variable is called error if the type is ResultUtils? is not better if you call this variable result?

Reply:
Done thx